### ISSUE TITLE
Add a dyslexic mode to the options menu that removes ALL CAPS and makes other improvements to help dyslexic players

### ISSUE THREAD

**Issue by FairlySadPanda** [space-wizards/space-station-14#40086]
## Description
Based on the idea of this well-intentioned PR

https://github.com/space-wizards/space-station-14/pull/39986

Add an option to the options menu that can be referred to when displaying text in the UI. This can then be used by UI in circumstances where ALL CAPS or other problematic ways of using text are used that are hard for dyslexic people to read.

With this option existant, review instances of ALL CAPS and other places where a dyslexic reader might struggle. Come up with ideas for fixes for each area.

Add sub-tasks to this issue that cover these emergent tasks, and then close them. Once all sub-tasks are closed, this issue can be  closed.

**Comment by slarticodefast** [comment]
Especially a setting for a dyslexic font that applies to all UIs should be added.
That should be a more general solution that just for the caps case.
